### PR TITLE
checkstyle WhiteAround

### PR DESCRIPTION
**Pull request by garyrussell**
WhiteAroundCheck script

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -69,6 +69,7 @@ subprojects { subproject ->
 	apply from:   "${rootDir}/src/checkstyle/fixModifiers.gradle"
 	apply from:   "${rootDir}/src/checkstyle/fixThis.gradle"
 	apply from:   "${rootDir}/src/checkstyle/fixRightCurly.gradle"
+	apply from:   "${rootDir}/src/checkstyle/fixWhiteAround.gradle"
```
Comment (artembilan) [inline build.gradle]: There is no such a script in the PR.
From other side it is so big, that I would like to see only the script and let you merge the final result on your own.

**Comment by garyrussell** [comment]
The script is in the commit - just that GH won't show it, 

    git log --stat

    ...
     src/checkstyle/checkstyle.xml                                                              |   4 +-
     src/checkstyle/fixWhiteAround.gradle                                                       |  64 +++++++++++++++++++++
     508 files changed, 2029 insertions(+), 1918 deletions(-)

I copied the script unchanged from spring-amqp; I had to do a few manual fixes afterwards, such as https://github.com/garyrussell/spring-integration/blob/checkstyle/spring-integration-amqp/src/test/java/org/springframework/integration/amqp/StubRabbitConnectionFactory.java#L335

where some anonymous inner classes were badly formatted.

I suggest we just wait for travis and merge if good.

**Comment by garyrussell** [comment]
Hmmm - looks like travis is not building it - bug? - I'll build locally

**Comment by artembilan** [comment]
H-m. What am I missing?
But I don't see any Travis progress here...

**Comment by artembilan** [comment]
Oh! You see the same :smile: 
Don't worry, I can do that on merge

**Comment by garyrussell** [comment]
BUILD SUCCESSFUL

Total time: 8 mins 11.286 secs

**Pull request merged**